apps/emqx_management/src/emqx_mgmt_auth.erl

```diff
@@ -365,6 +369,7 @@ init_bootstrap_file(File) ->
                     reason => Reason
```

Comment:
msg has been printed in the log. There should be no need to repeat it.
only return the format-reason but print explain-reason in log.

Reply:
No, most of the time this function is called at startup and will cause EMQX to crash, thus failing to print messages in the log file.